README.md

```diff
@@ -91,7 +91,7 @@ So you have now:
       type: beaglebone-black
       uart:
         idvendor: "0403"
-	idproduct: 6001
+	idproduct: "6001"
```

Comment:
No, please do not require quotes.  Instead requre (and document in the examples) to use a hex prefix 0x.